crates/astria-sequencer/src/bridge/component.rs

```diff
@@ -24,6 +24,7 @@ impl Component for BridgeComponent {
     async fn init_chain<S: StateWriteExt>(mut state: S, app_state: &Self::AppState) -> Result<()> {
         state.put_init_bridge_account_base_fee(app_state.fees.init_bridge_account_base_fee);
         state.put_bridge_lock_byte_cost_multiplier(app_state.fees.bridge_lock_byte_cost_multiplier);
+        state.put_bridge_sudo_change_fee(app_state.fees.bridge_sudo_change_fee);
```

Comment:
It seems like this will break if we don't have a fee set in genesis so node could not sync with old genesis?